src/components/clickablepicture/ClickablePicture.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import img from './maxence.png'
+import imgClicked from './maxence-glasses.png'
+
+export class ClickablePicture extends Component {
```

Comment:
Good job here, I would just change your state to true, to show as default the image without the glasses.